2022/2xxx/CVE-2022-2049.json

```diff
@@ -3,15 +3,80 @@
     "data_format": "MITRE",
```

Comment:
File name CVE ID and JSON CVE ID do not match.

Reply:
Now corrected